project_multifamily_beta/housing_characteristics/Geometry Wall Type.tsv

```diff
@@ -0,0 +1,91 @@
+CR05	2010s	0.422222	0.577778	0.000000	0.000000
+CR05	<1940	0.549168	0.450832	31.000000	338530.375000
+CR06	1940s	0.286792	0.713208	10.000000	108616.500000
+CR06	1950s	0.189793	0.810207	31.000000	336165.468750
+CR06	1960s	0.120356	0.879644	54.000000	577343.625000
+CR06	1970s	0.050390	0.949610	47.000000	490867.937500
+CR06	1980s	0.047698	0.952302	66.000000	667449.937500
+CR06	1990s	0.030284	0.969716	63.000000	682301.625000
+CR06	2000s	0.051870	0.948130	58.000000	585226.875000
+CR06	2010s	0.051870	0.948130	0.000000	0.000000
+CR06	<1940	0.144663	0.855337	38.000000	398488.593750
+CR07	1940s	0.632781	0.367219	104.000000	1058629.375000
+CR07	1950s	0.428523	0.571477	240.000000	2674402.750000
+CR07	1960s	0.462085	0.537915	186.000000	2022729.875000
+CR07	1970s	0.385163	0.614837	159.000000	1928237.000000
+CR07	1980s	0.325288	0.674712	118.000000	1424628.375000
+CR07	1990s	0.240546	0.759454	89.000000	1109901.750000
+CR07	2000s	0.252049	0.747951	98.000000	1248561.875000
+CR07	2010s	0.252049	0.747951	0.000000	0.000000
+CR07	<1940	0.502697	0.497303	334.000000	3792259.250000
+CR08	1940s	0.439453	0.560547	80.000000	678959.062500
+CR08	1950s	0.436076	0.563924	190.000000	1308334.625000
+CR08	1960s	0.530254	0.469746	212.000000	1551433.000000
+CR08	1970s	0.445075	0.554926	328.000000	2408617.750000
+CR08	1980s	0.316525	0.683475	255.000000	2111508.250000
+CR08	1990s	0.286744	0.713256	302.000000	2539058.500000
+CR08	2000s	0.356696	0.643304	307.000000	2289566.000000
+CR08	2010s	0.356696	0.643304	0.000000	0.000000
+CR08	<1940	0.452893	0.547107	206.000000	1550085.500000
+CR09	1940s	0.432119	0.567881	111.000000	1059438.875000
+CR09	1950s	0.471493	0.528507	286.000000	2745943.250000
+CR09	1960s	0.684412	0.315588	334.000000	3327770.250000
+CR09	1970s	0.660423	0.339577	514.000000	4968035.500000
+CR09	1980s	0.513308	0.486692	603.000000	5885774.500000
+CR09	1990s	0.448748	0.551252	573.000000	5974077.000000
+CR09	2000s	0.510205	0.489795	659.000000	6479666.500000
+CR09	2010s	0.510205	0.489795	0.000000	0.000000
+CR09	<1940	0.271623	0.728377	124.000000	1357769.250000
+CR10	1940s	1.000000	0.000000	2.000000	15948.900391
+CR10	1950s	0.743861	0.256139	18.000000	212219.953125
+CR10	1960s	0.720638	0.279362	18.000000	186049.875000
+CR10	1970s	0.714895	0.285105	81.000000	795593.187500
+CR10	1980s	0.718410	0.281590	71.000000	818635.500000
+CR10	1990s	0.776347	0.223653	89.000000	971213.437500
+CR10	2000s	0.903625	0.096375	83.000000	917632.000000
+CR10	2010s	0.903625	0.096375	0.000000	0.000000
+CR10	<1940	0.654133	0.345867	5.000000	67391.273438
+CR11	1940s	0.721746	0.278254	79.000000	601399.500000
+CR11	1950s	0.752700	0.247300	264.000000	1997365.750000
+CR11	1960s	0.808023	0.191977	262.000000	2005932.625000
+CR11	1970s	0.672620	0.327380	330.000000	2526129.500000
+CR11	1980s	0.702261	0.297739	232.000000	1781272.250000
+CR11	1990s	0.704148	0.295852	177.000000	1317830.875000
+CR11	2000s	0.832828	0.167172	129.000000	948427.625000
+CR11	2010s	0.832828	0.167172	0.000000	0.000000
```

Comment:
I'm a little bit suspicious of these high masonry numbers in CA. Presumably because they have a lot of stucco? Let's flag this somehow to revisit someday, perhaps by purchasing this:  "https://www.homeinnovation.com/trends_and_reports/data/new_construction/structural_systems_for_walls"

Reply:
It is quite high. Do you want to create an issue around it?